src/server/gc_worker/config.rs

```diff
@@ -18,6 +23,7 @@ pub struct GcConfig {
     pub batch_keys: usize,
     pub max_write_bytes_per_sec: ReadableSize,
     pub enable_compaction_filter: bool,
+    pub compaction_filter_minimal_version: String,
```

Comment:
Why is it a configure variable rather than a const variable?

Reply:
Some users can enable it before 5.0.0, if they will never fallback from 4.1 to 4.0.

Reply:
I thought we would never cherry-pick this PR to release-4.0 branch....
If we cherry-pick it to release-4.0, all users of 4.x will run GC-worker without raft-replication.

Reply:
At least I don't think it should be able to be configured in the config file. If we do need to cherry pick to 4.0 some day, we can submit a PR to update the constant.